code/modules/power/light_bulbs.dm

```diff
@@ -0,0 +1,128 @@
+// the light item
+// can be tube or bulb subtypes
+// will fit into empty /obj/machinery/light of the corresponding type
+
+/obj/item/light_bulb
+	icon = 'icons/obj/lighting.dmi'
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/inhands/equipment/lights_left.dmi',
+		slot_r_hand_str = 'icons/mob/inhands/equipment/lights_right.dmi',
+	)
+	force = 2
+	throwforce = 5
+	w_class = WEIGHT_CLASS_SMALL
+	var/status = 0		// LIGHT_OK, LIGHT_BURNED or LIGHT_BROKEN
+	var/bulb_type
```

Comment:
Is this used for anything you couldn't just initial(iconstate) instead?

Reply:
used base icon state